lib/NormandyDriver.js

```diff
@@ -124,5 +126,30 @@ exports.NormandyDriver = function(recipeRunner, sandbox, extraContext) {
     envExpression(expr) {
       return EnvExpressions.eval(expr, extraContext);
     },
+
+    showStudyConsentPage(options) {
+      tabs.open({
+        url: 'resource://shield-recipe-client/consent-page.html',
+        onReady: tab => {
+          const worker = tab.attach({
```

Comment:
This is surprising. Can we not use `<script>` tags in `consent-page.html`? Is this so we can attach the event handling to the worker?